content/kubermatic/master/advanced/addons/_index.en.md

```diff
@@ -55,12 +54,70 @@ To deploy the changes:
 helm upgrade --install --wait --timeout 300 --values values.yaml --namespace kubermatic kubermatic charts/kubermatic
 ```
 
-##### Setting a custom docker registry
+##### Setting a Custom Docker Registry
 
 In case you want to set a custom registry for all addons, you can specify the `-overwrideRegistry` flag on the `kubermatic-controller-manager` or via the helm setting `kubermatic.controller.overwriteRegistry`.
 It will set the specified registry on all control plane components & addons.
 
-### How to add a custom addon
+### Accessible Addons
+Accessible addons can be installed in each user-cluster in Kubermatic on user demand. If addon is both default
```

Comment:
If *an* addon is both default and accessible*,* then it will be installed in *the* user-cluster, but also *be* visible, who can manage it from the UI like *the* other accessible addons.